grafana/provisioning/dashboards/metrics.json

```diff
@@ -23,183 +23,187 @@
-                "x": 0,
-                "y": 1
-            },
-            "id": 6,
-            "options": {
-                "orientation": "auto",
-                "reduceOptions": {
-                    "calcs": [
-                        "lastNotNull"
-                    ],
-                    "fields": "",
-                    "values": false
-                },
-                "showThresholdLabels": false,
-                "showThresholdMarkers": false
-            },
-            "pluginVersion": "10.1.2",
-            "targets": [
+            "panels": [
                 {
                     "datasource": {
                         "type": "prometheus",
                         "uid": "PBFA97CFB590B2093"
                     },
-                    "editorMode": "code",
-                    "expr": "sum(jellyfish_rooms{instance=~\"$instance\"}) by(instance)",
-                    "instant": false,
-                    "legendFormat": "{{instance}}",
-                    "range": true,
-                    "refId": "A"
-                }
-            ],
-            "title": "Current number of rooms",
-            "type": "gauge"
-        },
-        {
-            "datasource": {
-                "type": "prometheus",
-                "uid": "${ds}"
-            },
-            "description": "Present the current number of rooms by jellyfish instance.",
-            "fieldConfig": {
-                "defaults": {
-                    "color": {
-                        "fixedColor": "blue",
-                        "mode": "continuous-BlYlRd"
-                    },
-                    "custom": {
-                        "axisCenteredZero": false,
-                        "axisColorMode": "text",
-                        "axisLabel": "",
-                        "axisPlacement": "auto",
-                        "barAlignment": 0,
-                        "drawStyle": "line",
-                        "fillOpacity": 10,
-                        "gradientMode": "none",
-                        "hideFrom": {
-                            "legend": false,
-                            "tooltip": false,
-                            "viz": false
-                        },
-                        "insertNulls": false,
-                        "lineInterpolation": "linear",
-                        "lineWidth": 1,
-                        "pointSize": 5,
-                        "scaleDistribution": {
-                            "type": "linear"
-                        },
-                        "showPoints": "never",
-                        "spanNulls": false,
-                        "stacking": {
-                            "group": "A",
-                            "mode": "none"
+                    "fieldConfig": {
+                        "defaults": {
+                            "color": {
+                                "fixedColor": "dark-blue",
+                                "mode": "shades"
```

Comment:
idk from where you got that, but I got an error:
Error: "shades" not found in: fixed,thresholds,palette-classic,continuous-GrYlRd,continuous-RdYlGr,continuous-BlYlRd,continuous-YlRd,continuous-BlPu,continuous-YlBl,continuous-blues,continuous-reds,continuous-greens,continuous-purples

trying to edit the panel. Deleting the `color` object from json view helped but I think it's something that should be addressed.

Also, for this panel you can use `Stat` visualization, not `Gauge` - I think it would go better.